docs/buildah-push.md

```diff
@@ -95,6 +99,10 @@ This example extracts the imageID image to a container registry named registry.e
 
  `# buildah push imageID docker://registry.example.com/repository:tag`
 
+This example extracts the imageID image to a container registry named registry.example.com and saves the digest in the specified digestfile.
```

Comment:
One more non-blocking comment: The “extracts” verb is used in all examples, and does not really match in _any_ of them.

At least this new one should probably say “pushes”.

Reply:
Yeah I noted that this weekend and wasn't happy with "extracts" either.  But didn't come up with a better word quickly so just kept this one in line with the others.

Reply:
reworked that too in the latest commit.